test/acceptance/spec/20-details.js

```diff
@@ -27,7 +27,8 @@ describe('Details Page', () => {
       browserText[6].getText().should.match(new RegExp('Place of birth *' + record.child.birthplace));
       browserText[8].getText().should.match(new RegExp('Name *' + record.mother.name.fullName));
       browserText[9].getText().should.match(new RegExp('Maiden name *' + record.mother.maidenSurname));
-      browserText[10].getText().should.match(new RegExp('Previous marriage name *' + record.mother.marriageSurname));
+      browserText[10].getText().should.match(new RegExp(
+        'Surname at marriage\\sif different from maiden name *' + record.mother.marriageSurname));
```

Comment:
so you want?:
```
Surname at marriage if
different from maiden
name```

Reply:
Yes, I think so.

That's almost what is written on the certificate except they use the phrase "maiden surname" whereas we already have "maiden name" elsewhere.